installer/darwin/dmgconf.py

```diff
@@ -31,4 +31,5 @@ def get_size(start_path = '.'):
 #background = 'builtin-arrow'
 default_view = 'icon-view'
 
-
+format = 'UDZO'
+license = {'licenses': {'en_US': 'LICENSE.txt'}}
```

Comment:
Clearly it's not a serious issue, but the [dmgbuild docs](https://dmgbuild.readthedocs.io/en/latest/settings.html#license-settings) say that a `'default-language'` key is also required in the `license` dictionary.

Also, is it okay that `LICENSE.txt` is relative to the top-level and not installer/darwin/dmgconf.py ?

Reply:
> Clearly it's not a serious issue, but the dmgbuild docs say that a 'default-language' key is also required in the license dictionary.

Ah, good catch!  Since it is required, patched this in e96ed0c0.

> Also, is it okay that LICENSE.txt is relative to the top-level and not installer/darwin/dmgconf.py ?

Based on what's in the rest of `dmgconf.py`, I'd be inclined to think so ... but just in case, I've modified the configuration program to use the file's path for the DMG badge icon and the license file in d849a071.

Reply:
Just following up on this, turns out that `__file__` is not available within the `dmgconf.py` so I've updated the filepath variable to clarify intent and added a few comments to clarify that this is all assumed to be executed from the project root.  Patched in da8296a1.

As for a DMG, the latest binary run is still testing, but here's the actions build from e96ed0c that should allow you to download and use the dmg artifact directly from github actions: https://github.com/natcap/invest/runs/1476979402  That'll be the best thing to test!